Automattic/wp-calypso
# Add checkout included features list for new Stats products

This PR adds the list of "Included features" displayed on the checkout page, for the new Stats products. (See screenshot):

#### Screenshots:

**Stats paid products (Commercial and PWYW):**

![Markup 2023-06-30 at 18 54 08](https://github.com/Automattic/wp-calypso/assets/11078128/ab54e4f8-cb01-4052-9655-0de901702643)

![Markup 2023-06-30 at 19 00 31](https://github.com/Automattic/wp-calypso/assets/11078128/edb1cf37-a866-4cc2-824d-84c207835f7c)

** Stats Free product ($0): **

![Markup 2023-06-30 at 18 58 42](https://github.com/Automattic/wp-calypso/assets/11078128/069c69d2-ffba-4430-a3ca-426a162fd00c)

Stats "Included features" items discussion: p1HpG7-ngN-p2#comment-64474
Asana task: 1201210512993648-as-1204939720795290/f

## Proposed Changes

- Adds the new Jetpack Stats products constants to the `calypso-products` package.
- Add the "Included features" list for Jetpack Stats products on the Checkout page.
- Update unit tests.

## Testing Instructions

- Checkout this PR & spin up Calypso blue (`git fetch && git checkout add/checkout-stats-included-features && yarn start`)
- Go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_stats_monthly
- Verify the Stats (Commercial) product is in the cart and the "Included features" list items are displaying and look like the screenshots above.
- Remove the product from the cart (unless code-D115101 is already deployed, which prevents concurrent products in cart).
- Go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_stats_pwyw_yearly:-q-12
- Verify the Stats (Paid) product is in the cart for $12 dollars and the "Included features" list items are displaying and look like the screenshots above.
- Remove the product from the cart (unless code-D115101 is already deployed, which prevents concurrent products in cart).
- Go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_stats_free_yearly
- Verify the Stats (Free non-commercial license) product is in the cart and there are only 2 "Included features" list items and they look like the screenshots above.
- 
## Pre-merge Checklist
